Index: toBeDeleted/forPatch.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- toBeDeleted/forPatch.go	(date 1511780928000)
+++ toBeDeleted/forPatch.go	(date 1511780967365)
@@ -6,4 +6,8 @@
 
 func createPath()  {
 	fmt.Println("Create patch")
+}
+
+func change1()  {
+	fmt.Print("change 1")
 }
\ No newline at end of file
